src/backend/base/langflow/components/tools/mcp_sse.py

```diff
@@ -89,6 +98,7 @@ async def build_output(self) -> list[Tool]:
                 Tool(
```

Comment:
Maybe we should use StructuredTool (support zero parameter tool)
`from langchain_core.tools import StructuredTool`

Reply:
`src/backend/base/langflow/base/mcp/util.py` need to be modify to support zero parameter tool, maybe create another PR after this been merged